src/pal/src/include/pal/file.h

```diff
@@ -42,6 +42,14 @@ typedef struct _find_handle
     char   **next;
 } find_obj;
 
+#ifndef __ANDROID__
```

Comment:
I would prefer putting it into the palinternal.h somewhere close to where PROCESS_PIPE_NAME_PREFIX is defined. The file.h is a header file for file.cpp, and this is more general. Then you don't need to add any additional includes.